### ISSUE TITLE
Initial checkout fails in latest version (1.3.1)

### ISSUE THREAD

**Issue by tony** [vcs-python/vcspull#298]
```
|django-rq-dashboard| (git)  Adding remote origin <git+https://github.com/brutasse/django-rq-dashboard.git>
fatal: remote origin already exists.
Traceback (most recent call last):
  File "/home/t/.local/bin/vcspull", line 8, in <module>
    sys.exit(cli.cli())
  File "/home/t/.local/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/t/.local/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/t/.local/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/t/.local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/t/.local/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/t/.local/lib/python3.7/site-packages/vcspull/cli.py", line 117, in update
    list(map(update_repo, found_repos))
  File "/home/t/.local/lib/python3.7/site-packages/vcspull/cli.py", line 159, in update_repo
    r.update_repo()  # Creates repo if not exists and fetches
  File "/home/t/.local/lib/python3.7/site-packages/libvcs/git.py", line 139, in update_repo
    self.obtain()
  File "/home/t/.local/lib/python3.7/site-packages/libvcs/git.py", line 127, in obtain
    self.remote_set(name=r['remote_name'], url=r['url'])
  File "/home/t/.local/lib/python3.7/site-packages/libvcs/git.py", line 331, in remote_set
    self.run(['remote', 'add', name, url])
  File "/home/t/.local/lib/python3.7/site-packages/libvcs/base.py", line 108, in run
    cwd=cwd,
  File "/home/t/.local/lib/python3.7/site-packages/libvcs/util.py", line 178, in run
    raise exc.CommandError(output=output, returncode=code, cmd=cmd)
libvcs.exc.CommandError: Command failed with code 128: git remote add origin https://github.com/brutasse/django-rq-dashboard.git
```

**Comment by tony** [comment]
It's actually broke:

```
django-rq-dashboard:
    repo: 'git+https://github.com/brutasse/django-rq-dashboard.git'
    remotes:
      eduflow: 'git+ssh://git@github.com/eduflow/django-rq-dashboard.git'
      sbussetti: 'git+ssh://git@github.com/sbussetti/django-rq-dashboard.git'
      morlandi: 'git+ssh://git@github.com/morlandi/django-rq-dashboard.git'
```

```
eduflow git+ssh://git@github.com/eduflow/django-rq-dashboard.git (fetch)
eduflow git+ssh://git@github.com/eduflow/django-rq-dashboard.git (push)
origin  https://github.com/brutasse/django-rq-dashboard.git (fetch)
origin  https://github.com/brutasse/django-rq-dashboard.git (push)
```

**Comment by tony** [comment]
Fixed in #300
